doc/moveit_grasps/moveit_grasps_tutorial.rst

```diff
@@ -0,0 +1,121 @@
+MoveIt Grasps
+=======================
+
+MoveIt Grasps is a grasp generator for objects such as blocks or cylinders for use with the MoveIt pick and place pipeline. It also contains its own components for filtering grasps based on reachability and approach/retreat motions.
+The grasp generation algorithm is based on simple cuboid shapes and does not consider friction cones or other grasp dynamics.
+
+Its current implementation takes as input a pose of a cuboid, and the cuboid's size, and generates a large number of potential grasp approaches and directions.
+Also included is a grasp filter for removing kinematically infeasible grasps via threaded IK solvers.
+
+.. image:: moveit_grasps.png
+   :width: 500pt
+   
+Installing MoveIt Grasps
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Install From Source
+--------------------
+
+Clone the `moveit_grasps <https://github.com/ros-planning/moveit_grasps>`_ repository into a `catkin workspace <https://ros-planning.github.io/moveit_tutorials/doc/getting_started/getting_started.html#create-a-catkin-workspace>`_. For this tutorial we use Franka Emika's Panda robot setup from `panda_moveit_config <https://github.com/ros-planning/panda_moveit_config>`_::
+
+    git clone https://github.com/ros-planning/moveit_grasps
+
+Use the rosdep tool to automatically install its dependencies::
+
+    rosdep install --from-paths src --ignore-src --rosdistro kinetic
+    
+Build the workspace::
+
+    catkin build
+
+Setup
+^^^^^^^^^^^^^^^^
+
+Robot-Agnostic Configuration
+----------------------------
+
+You will first need a configuration file that described your robot's end effector geometry.
+Currently an example format can be seen in this repository at `config_robot/panda_grasp_data.yaml <https://github.com/ros-planning/moveit_grasps/blob/kinetic-devel/config_robot/panda_grasp_data.yaml>`_.
+See the comments within that file for explanations.
+
+To load that file at launch, you copy the example in the file `launch/load_panda.launch <https://github.com/ros-planning/moveit_grasps/blob/kinetic-devel/launch/load_panda.launch>`_ where you should see the line::
+
+    ...
+    
+    <rosparam command="load" file="$(find moveit_grasps)/config_robot/panda_grasp_data.yaml"/>
+    
+    ...
+
+Within that file you will find all of the gripper specific parameters necessary for customizing MoveIt Grasps with any suction or finger gripper.
```

Comment:
From looking at the parameter file, the meaning of all those parameters doesn't yet become clear.
I guess the code is limited to parallel-jaw grippers, right? Why, for example, is it required to define the finger width?

Reply:
I added further parameter description to the example config https://github.com/ros-planning/moveit_grasps/pull/36. I agree that some of the parameters seem a bit confusing at first but I think the comments should clarify all of them.